WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -888,7 +888,6 @@ extension WordPressAppDelegate {
         WPStyleGuide.configureLightNavigationBarAppearance()
         WPStyleGuide.configureToolbarAppearance()
 
-        UISegmentedControl.appearance().setTitleTextAttributes( [NSAttributedString.Key.font: WPStyleGuide.regularTextFont()], for: .normal)
```

Comment:
This was not needed anymore in the app but it was breaking the design of some the network logger screens.

Reply:
From a quick search, I see [one place in Stats](https://github.com/wordpress-mobile/WordPress-iOS/blob/23.7.0.0/WordPress/Classes/ViewRelated/Stats/Insights/ViewsVisitors/ViewsVisitorsLineChartCell.swift#L135) uses `UISegmentedControl`. Have you checked whether removing this line would impact that Stats UI?

Reply:
Yes, it overwrites the text attributes, so this change doesn't affect it.

<img width="453" alt="Screenshot 2023-11-22 at 5 38 46 PM" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/a5fa4031-7010-4ce9-8d07-0f9fb2b5f412">